cloud/scope/machine.go

```diff
@@ -560,7 +560,7 @@ func (m *MachineScope) ReconcileCreateInstanceOnLB(ctx context.Context) error {
 			m.OCIMachine.Status.CreateBackendWorkRequestId = *resp.OpcWorkRequestId
 			logger.Info("Add instance to LB backend-set", "WorkRequestId", resp.OpcWorkRequestId)
 			logger.Info("Waiting for LB work request to be complete")
-			_, err = ociutil.AwaitLBWorkRequest(ctx, m.LoadBalancerClient, m.WorkRequestsClient, resp.OpcWorkRequestId)
+			_, err = ociutil.AwaitLBWorkRequest(ctx, m.LoadBalancerClient, resp.OpcWorkRequestId)
```

Comment:
Is the `WorkRequestsClient` still needed in machine then?

Reply:
Yes, this is required to fetch the work-request errors in case of ocimachine objects. So when we provision a compute resource and some work-request-errors are there, this client is required to fetch those using workRequestId, unlike LB and NLB which have the required errorDetails implementation in their own services.